app/controllers/peer_reviews_controller.rb

```diff
@@ -157,8 +157,15 @@ def assign_groups
         return
       end
     when 'unassign'
-      deleted_count, undeleted_reviews = PeerReview.unassign(selected_reviewee_group_ids,
-                                                             reviewers_to_remove_from_reviewees_map)
+      peer_reviews_filtered = PeerReview.joins(:reviewer, :reviewee)
+                                        .where(reviewer: { id: selected_reviewer_group_ids })
+                                        .where(reviewee: { id: selected_reviewee_group_ids })
```

Comment:
Also, these don't need to be separate `where` clauses, as `.where` takes an arbitrary number of arguments and treats them as an `AND`, which is what you want in this case